Update Ray integration docs (#13385)
PrefectHQ/prefect: docs/integrations/prefect-ray/index.md

# prefect-ray

> [Ray](https://docs.ray.io/en/latest/index.html) helps you run your flow in parallel and distribute runs over multiple machines.

Reviewer: ```suggestion
[Ray](https://docs.ray.io/en/latest/index.html) can run the tasks in your flow in parallel by distributing them over multiple machines.
```
